### PR TITLE
Cleaned up data models.

### PR DESCRIPTION
**Pull request by mjl757**
Removed unnecessary Repository files.
Removed unnecessary id fields on linker tables.
Added Constructors.

### CODE REVIEW

File: family2family/src/main/java/org/cbtf/f2f/model/Diagnosis.java [hunk 1]
```diff
@@ -23,10 +23,18 @@
 
     private String description;
 
+    public Diagnosis(String description) {
```
Comment (jwest011) [inline family2family/src/main/java/org/cbtf/f2f/model/Diagnosis.java:26]: http://localhost:8080/internal-api/Diagnosis errors. Complains about not having a default constructor.

File: family2family/src/main/java/org/cbtf/f2f/model/Subcategory.java [hunk 2]
```diff
@@ -34,8 +34,11 @@
     @JoinColumn(name = Category.COLUMN_CATEGORY_ID)
     private Category category;
 
-    @OneToMany(mappedBy = ChildSubcategory.FIELD_SUBCATEGORY)
-    private List<ChildSubcategory> childSubcategories;
+    public Subcategory(String description, Boolean active, Category category) {
```
Comment (jwest011) [inline family2family/src/main/java/org/cbtf/f2f/model/Subcategory.java]: http://localhost:8080/internal-api/Subcategory errors. Complains about not having a default constructor.

File: family2family/src/main/java/org/cbtf/f2f/model/Category.java [hunk 3]
```diff
@@ -25,12 +25,15 @@
 
     private Boolean active;
 
+    public Category(String description, Boolean active, List<Subcategory> subCategories) {
```
Comment (jwest011) [inline family2family/src/main/java/org/cbtf/f2f/model/Category.java]: http://localhost:8080/internal-api/Category errors. Complains about not having a default constructor.

File: family2family/src/main/java/org/cbtf/f2f/model/Users.java [hunk 4]
```diff
@@ -15,6 +17,17 @@
 
     private Boolean active;
 
+    @OneToOne
+    @JoinColumn(name = Contact.COLUMN_PRIMARY_EMAIL)
+    private Contact contact;
+
+    public Users(String id, Boolean admin, Boolean active, Contact contact) {
```
Comment (jwest011) [inline family2family/src/main/java/org/cbtf/f2f/model/Users.java]: http://localhost:8080/internal-api/Users errors. Complains about not having a default constructor.

**Review by jwest011 — CHANGES_REQUESTED** [review]
A few of the internal-api services are failing.

File: family2family/src/main/java/org/cbtf/f2f/model/Diagnosis.java (hunk 1 above)
Comment (mjl757) [reply family2family/src/main/java/org/cbtf/f2f/model/Diagnosis.java:26]: I'll add Default constructors to all the classes.

File: family2family/src/main/java/org/cbtf/f2f/model/ChildDiagnosisRepository.java [hunk 5]
```diff
@@ -1,10 +0,0 @@
-package org.cbtf.f2f.model;
```
Comment (mjl757) [inline family2family/src/main/java/org/cbtf/f2f/model/ChildDiagnosisRepository.java]: I realized that while these classes are not needed for showing the table in the console, they are needed for displaying the tables on http://localhost:8080/internal-api .

**Review by sadhnaahuja — APPROVED** [review]

**Review by jwest011 — APPROVED** [review]

**Pull request merged**